src/main/java/frc/team2412/robot/util/motorcontroller/MotorController.java

```diff
@@ -0,0 +1,105 @@
+package frc.team2412.robot.util.motorcontroller;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.revrobotics.CANSparkMax.ControlType;
+import com.revrobotics.CANSparkMax.IdleMode;
+
+public abstract class MotorController {
+	public enum MotorNeutralMode {
+		COAST {
+			@Override
+			public NeutralMode getCTRE() {
+				return NeutralMode.Coast;
+			}
+
+			@Override
+			public IdleMode getREV() {
+				return IdleMode.kCoast;
+			}
+		},
+		BRAKE {
+			@Override
+			public NeutralMode getCTRE() {
+				return NeutralMode.Brake;
+			}
+
+			@Override
+			public IdleMode getREV() {
+				return IdleMode.kBrake;
+			}
+		};
+
+		public NeutralMode getCTRE() {
+			return NeutralMode.Coast;
+		}
+
+		public IdleMode getREV() {
+			return IdleMode.kCoast;
+		}
+	}
```

Comment:
Fine as it is, but we can be a bit more concise and readable (similar thing for `ControlMode`):
```suggestion
	public enum MotorNeutralMode {
		COAST(NeutralMode.Coast, IdleMode.kCoast),
		BRAKE(NeutralMode.Brake, IdleMode.kBrake);
		
		private final NeutralMode ctre;
		private final IdleMode rev;
		
		MotorNeutralMode(NeutralMode ctre, IdleMode rev) {
		    this.ctre = ctre;
		    this.rev = rev;
		}

		public NeutralMode getCTRE() {
			return ctre;
		}

		public IdleMode getREV() {
			return rev;
		}
	}
```